doc/changes/latest.rst

```diff
@@ -60,3 +60,4 @@ Changes
 - Refactor error raising tests using context managers (:gh:`3854` BY `François Paugam`_)
 - Added warning to deprecate `darkness` in ``surf_plotting._compute_facecolors_matplotlib`` and ``html_surface._get_vertexcolor`` (:gh`3855` by `Alisha Kodibagkar`_)
 - :bdg-secondary:`Doc` Replace skipped doctests with default code-blocks (:gh:`3681` in by `Patrick Sadil`_)
+- Move the :func:`~nilearn.plotting.html_surface._mix_colormaps` in :mod:`~nilearn.plotting.cm` (:gh:`3919` by `Ahmad Chamma`_)
```

Comment:
The build of the doc is getting the following warnings:

```
/home/runner/work/nilearn/nilearn/doc/changes/latest.rst:63: WARNING: py:func reference target not found: nilearn.plotting.html_surface._mix_colormaps
/home/runner/work/nilearn/nilearn/doc/changes/latest.rst:63: WARNING: py:mod reference target not found: nilearn.plotting.cm
```

The first one makes sense as the function does not exist anymore, so maybe remove the `:func:`.
Not sure about the second warning.

Reply:
`_mix_colormaps` is private and not exposed so you can't link to it. Just do something like `` `_mix_colormaps` ``. To be explicit just mention the filenames you moved it from/to. `mod` role will work on the level of the modules like `plotting`, `maskers` etc. but not to link the python scripts inside those modules. Just use `` `cm.py` `` and be explicit that it's in the plotting module.